packages/maskbook/src/_locales/zh/messages.json

```
@@ -128,565 +128,565 @@
-    "plugin_wallet_transaction_reverted": "",
+    "your_message": "你的訊息…",
+    "input_length_exceed_prompt": "{{name}} 的長度需小於 {{length}}。",
+    "relative_time_days_ago": "{{days}} 天前",
+    "relative_time_hours_ago": "{{hours}} 小時前",
+    "relative_time_minutes_ago": "{{minutes}} 分鐘前",
+    "relative_time_months_ago": "{{months}} 月前",
+    "relative_time_seconds_ago": "{{seconds}} 秒前",
+    "relative_time_years_ago": "{{years}} 年前",
+    "plugin_powered_by": "提供者為 ",
+    "plugin_wallet_select_a_token": "選擇一個代幣",
+    "plugin_wallet_connect_a_wallet": "連接到錢包",
+    "plugin_wallet_unlock_metamask": "解鎖MetaMask",
+    "plugin_wallet_no_gas_fee": "沒有Gas費用",
+    "plugin_wallet_update_gas_fee": "正在更新Gas費用…",
+    "plugin_wallet_invalid_network": "錯誤的網路",
+    "plugin_wallet_select_a_wallet": "選擇一個錢包",
+    "plugin_wallet_transaction": "交易",
+    "plugin_wallet_transaction_wait_for_confirmation": "等待確認中…",
+    "plugin_wallet_transaction_submitted": "你的交易已經被送出！",
+    "plugin_wallet_transaction_confirmed": "你的交易已經被確認！",
+    "plugin_wallet_transaction_reverted": "交易被復原！",
     "plugin_wallet_transaction_rejected": "交易被拒絕！",
-    "plugin_wallet_transaction_server_error": "",
-    "plugin_wallet_view_on_explorer": "在 Explorer 上查看",
-    "plugin_wallet_wrong_network_tip": "",
-    "plugin_wallet_on_create": "創建錢包",
+    "plugin_wallet_transaction_server_error": "交易由於內部JSON-RPC伺服器錯誤而失敗。",
+    "plugin_wallet_view_on_explorer": "在區塊鏈瀏覽器上查看",
+    "plugin_wallet_wrong_network_tip": "請連接到適當的乙太坊網絡。",
+    "plugin_wallet_on_create": "建立錢包",
     "plugin_wallet_on_connect": "連接錢包",
-    "plugin_wallet_setup_title_create": "",
-    "plugin_wallet_setup_description_words": "",
-    "plugin_wallet_setup_description_verify": "",
-    "plugin_wallet_setup_back": "",
-    "plugin_wallet_setup_create": "",
-    "plugin_wallet_setup_next": "",
-    "plugin_wallet_setup_verify": "",
+    "plugin_wallet_setup_title_create": "建立一個錢包",
+    "plugin_wallet_setup_description_words": "你的秘密備份短語將使備份和復原你的帳戶變得很容易。把這個短語寫在一張紙上，放在安全的地方。如果你想更加安全，把它寫在多張紙上，分別存放在2-3個不同的地方。",
+    "plugin_wallet_setup_description_verify": "請填入每個空格，以確保它是正確的。",
+    "plugin_wallet_setup_back": "上一步",
+    "plugin_wallet_setup_create": "建立",
+    "plugin_wallet_setup_next": "下一步",
+    "plugin_wallet_setup_verify": "驗證",
     "plugin_wallet_select_provider_dialog_title": "連接錢包",
-    "plugin_wallet_qr_code_with_wallet_connect": "使用 WalletConnect 兼容的錢包掃描 QR 碼",
-    "plugin_wallet_metamask_unlock": "解鎖 MetaMask",
-    "plugin_wallet_token_unlock": "",
-    "plugin_wallet_token_infinite_unlock": "",
-    "plugin_wallet_connect_dialog_title": "",
-    "plugin_wallet_connect_to_mask": "使用 Mask 創建",
-    "plugin_wallet_connect_to_metamask": "連接到 MetaMask",
-    "plugin_wallet_connect_to_walletconnect_on_web": "掃描以連接 WalletConnect",
-    "plugin_wallet_connect_to_walletconnect_on_app": "連接 WalletConnect",
+    "plugin_wallet_qr_code_with_wallet_connect": "使用一個與WalletConnect相容的錢包掃描QR code",
```

I personally wish to add a space between Chinese and English

e.g: 
```suggestion
    "plugin_wallet_qr_code_with_wallet_connect": "使用一個與 WalletConnect 相容的錢包掃描 QR Code",
```

---

fixed via 67aadfb